patches/chromium/fix_crash_on_nativetheme_change_during_context_menu_close.patch

```diff
@@ -1,42 +0,0 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Shelley Vohr <shelley.vohr@gmail.com>
```

Comment:
Looks like this was removed due to https://chromium-review.googlesource.com/c/chromium/src/+/4762986